recipe/meta.yaml

```diff
@@ -1,23 +1,23 @@
-{% set version = "3.0.6" %}
+{% set version = "4.0.1" %}
 {% set cuda_major = (cuda_compiler_version|default("12.0.0")).split(".")[0] %}
-{% set platform = "x86_64" %}  # [linux64]
-{% set platform = "SBSA" %}  # [aarch64]
-{% set platform = "windows" %}  # [win]
-{% set extension = "tgz" %}  # [not win]
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set extension = "tar.gz" %}  # [not win]
 {% set extension = "zip" %}  # [win]
 
 package:
   name: nvcomp
   version: {{ version }}
 
 source:
-  - url: https://developer.download.nvidia.com/compute/nvcomp/{{ version }}/local_installers/nvcomp_{{ version }}_{{ platform }}_{{ cuda_major }}.x.{{ extension }}
-    sha256: 2631c4cb36c2b65e1c8d3a818bc470b849893e72bdbee0365d22dc3f7c130954  # [linux64 and (cuda_compiler_version or "None").startswith("11")]
-    sha256: a76511cc55a40aa9f8ca6b6b437102cc90969c69bfbcd66265f2e04f8f01435f  # [linux64 and (cuda_compiler_version or "None").startswith("12")]
-    sha256: 21b0b6ad262101a9d89bc5dc83cddf3da6b57b0a5ef8d75e5876e921ca46c9c5  # [aarch64 and (cuda_compiler_version or "None").startswith("11")]
-    sha256: d1a25754c21e37e643ebe5828ffdb11d0f237e27c4f29b83b5c17de2e44247d5  # [aarch64 and (cuda_compiler_version or "None").startswith("12")]
-    sha256: 10b98c8b58b9186656bfaf06ce8a7d06a5ce3c1318a9eaa254c47cadb71f748d  # [win and (cuda_compiler_version or "None").startswith("11")]
-    sha256: f7f25250d628e015854c0f65688921a2947f53f8b72aa01ce7c45aa2ea04893c  # [win and (cuda_compiler_version or "None").startswith("12")]
+  - url: https://developer.download.nvidia.com/compute/nvcomp/{{ version }}/local_installers/nvcomp-{{ platform }}-{{ version }}-cuda{{ cuda_major }}.x.{{ extension }}  # [linux64 or aarch64 or win]
+    sha256: 8e92afa5f007e0d5373bf373ffa6d3c531e0a1d33c76f475e100bb0252e53777  # [linux64 and (cuda_compiler_version or "None").startswith("11")]
+    sha256: 609f64607d02e77c77c1848f81a04bbb1389329fa95533813c3fe7d77c57bd30  # [linux64 and (cuda_compiler_version or "None").startswith("12")]
+    sha256: 456129c03bfce6c3935b9f68455f40b65bc1b88b3f7972f70cc6fb9df222e676  # [aarch64 and (cuda_compiler_version or "None").startswith("11")]
+    sha256: bc314da00094fd93adbaf0bd762ab9f914643bd832ae45e238f321ee6477a2e0  # [aarch64 and (cuda_compiler_version or "None").startswith("12")]
+    sha256: 9263cfd57bb1ccec26e13b65fa2cd99c04877d54e42c17c57053bc5391ec733f  # [win and (cuda_compiler_version or "None").startswith("11")]
+    sha256: 4343aabbb4b3cf96bfbbb01ce2177cfdb3a10e477cf07702500ea98ff9e51308  # [win and (cuda_compiler_version or "None").startswith("12")]
```

Comment:
Seeing some checksum failures on CI. Can we please double check these?

Reply:
Ah sorry this is the license file checksum below that needs an update

<details>

```python
Downloading https://developer.download.nvidia.com/compute/cuda/redist/cuda_cudart/LICENSE.txt
Traceback (most recent call last):
  File "/opt/conda/bin/conda-build", line 11, in <module>
    sys.exit(execute())
  File "/opt/conda/lib/python3.10/site-packages/conda_build/cli/main_build.py", line 589, in execute
    api.build(
  File "/opt/conda/lib/python3.10/site-packages/conda_build/api.py", line 209, in build
    return build_tree(
  File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 3712, in build_tree
    packages_from_this = build(
  File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 2449, in build
    try_download(m, no_download_source=False, raise_error=True)
  File "/opt/conda/lib/python3.10/site-packages/conda_build/render.py", line 762, in try_download
    source.provide(metadata)
  File "/opt/conda/lib/python3.10/site-packages/conda_build/source.py", line 1035, in provide
    unpack(
  File "/opt/conda/lib/python3.10/site-packages/conda_build/source.py", line 169, in unpack
    src_path, unhashed_fn = download_to_cache(
  File "/opt/conda/lib/python3.10/site-packages/conda_build/source.py", line 125, in download_to_cache
    raise RuntimeError(
RuntimeError: SHA256 mismatch: 'e2c71babfd18a8e69542dd7e9ca018f9caa438094001a58e6bc4d8c999bf0d07' != '5db25d4fd138013b563f9a3d1d87f7de7df1dac014fd4cccdfbb3435a5cff761'
```

</details>

Reply:
Hmm. Yes, I’ll update.